cmd/completion/main.go

```diff
@@ -96,8 +97,10 @@ func main() {
 	cosignDockerMediaTypesOverrides := mapKeyValueArgs(cosignDockerMediaTypes)
 
 	ctx := context.Background()
-	if err := cosignSigner.Sign(ctx, report, cosignSecretLocation, annotations, cosignRepositoryOverrides, cosignDockerMediaTypesOverrides); err != nil {
-		logger.Fatalf("cosignSigner sign: %v\n", err)
+	if _, err := os.Stat(cosignSecretLocation); !os.IsNotExist(err) {
```

Comment:
Can we update the signer to exit with an error now? https://github.com/pivotal/kpack/pull/817/files#diff-2b751ca77925350f424134c6ca29134ce0cb23dff9b4439c90dffdfcd27c407eR38